omod/src/main/java/org/openmrs/module/appointments/web/contract/AppointmentDefaultResponse.java

```diff
@@ -176,4 +178,20 @@ public void setTeleconsultationLink(String teleconsultationLink) {
 	public String getTeleconsultationLink() {
 		return teleconsultationLink;
 	}
+
+	public void setPriority(String priority) {
```

Comment:
Why are getters/setters for Appointment Priority using String, instead of Enum?

Reply:
This is response contract class and we followed the same as that of appointment status